frontend/src/components/navigation/NavigationItem.module.scss

```diff
@@ -1,69 +1,65 @@
-@import 'styles/colors';
-@import 'styles/typography';
-@import 'styles/spacings';
+@import "styles/colors";
+@import "styles/typography";
+@import "styles/spacings";
 
 .root {
   .navLink {
     display: flex;
     color: $light-grey;
-    font-size: $font-size-big;
+    font-size: 15px;
     text-decoration: none;
     align-items: center;
+    flex-direction: column;
+    gap: 6px;
+    line-height: 19px;
+    padding: 11px 45px 14px 45px;
+    border-right: 1px solid $background;
 
-    & .iconWrapper {
-      text-align: center;
-      margin-bottom: 6px;
-
-      & > svg {
-        fill: $light-grey;
-      }
+    @media (max-width: 1500px) {
+      padding: 11px 23px 14px 23px;
     }
 
     &:hover {
       opacity: 0.9;
     }
 
-    & > div:first-child::after {
-      content: "";
-      position: relative;
-      display: block;
-      top: 16px;
-      left: calc(50% - 27.5%);
-      width: 55%;
-      height: 3px;
-    }
-
     &.active {
       color: $yellow;
-
-      .iconWrapper {
-        svg {
-          fill: $yellow;
-        }
-      }
-
-      & > div:first-child::after {
-        background-color: $yellow;
-      }
+      background: #4e516b;
     }
 
     & > div:first-child {
       margin-right: $spacing-base;
     }
 
-    & > div:last-child {
-      margin-top: 30px;
-      background-color: $blue;
-      font-size: $font-size-xxsmall;
-      line-height: calc(#{$font-size-xxsmall} + 4px);
-      color: $color-main-text;
-      border-radius: 8px;
-      height: 15px;
-      width: 28px;
-      vertical-align: center;
-      text-align: center;
-      font-weight: bold;
+    .counters {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      gap: 5px;
+
+      > * {
+        font-size: $font-size-xxsmall;
+        line-height: calc(#{$font-size-xxsmall} + 2px);
+        color: $color-main-text;
+        border-radius: 8px;
+        text-align: center;
+        font-weight: 400;
+        padding: 1px 5px;
```

Comment:
Extra minor tweak:

The design assumes that the number in this component could be large, but in reality it won't be. Because of that I think this count badge looks just a bit too small (to me).

I'm thinking the numbers in this badge could be just a bit more readable and maybe also look nicer (?) if we would increase the font-size + padding just a bit.

**Before:**
![Screenshot 2022-09-06 at 02 22 37](https://user-images.githubusercontent.com/36109955/188523199-099a048d-761e-4ec1-92fc-24338c7e1221.png)

**After:**
![Screenshot 2022-09-06 at 02 22 13](https://user-images.githubusercontent.com/36109955/188523174-ca5f68ae-832c-4a6b-bf27-57c45d773a3c.png)

Settings that I used in this example:
![Screenshot 2022-09-06 at 02 26 07](https://user-images.githubusercontent.com/36109955/188523377-6ab82403-aae4-4d0e-acd2-8d2ceb6af55f.png)

What do you think? I know this is very picky, but I think I noticed those small numbers right away and I'm trying to think of a solution that fixes that.

Reply:
Yep i also agree but i made it per design and kinda left it, good eye!